components/menu-item/style.scss

```diff
@@ -7,8 +7,9 @@
 	color: $dark-gray-500;
 
 	.dashicon,
-	.components-menu-items__item-icon {
-		margin-right: 5px;
+	.components-menu-items__item-icon,
+	> span > svg { // Also target plugin icons that can have arbitrary classes.
```

Comment:
This is a weird spot for a comment, can it go above the selector instead?

It doesn't need the "also" either (the comma implies that).

`// Target plugin icons that can have arbitrary classes by using an aggressive selector.` should do the trick!